OpenRA.Mods.Common/Widgets/ViewportControllerWidget.cs

```diff
@@ -191,7 +191,7 @@ public void UpdateMouseover()
 
 			if (underCursor != null)
 			{
-				ActorTooltip = underCursor.TraitsImplementing<ITooltip>().First();
+				ActorTooltip = underCursor.TraitsImplementing<ITooltip>().First(Exts.IsTraitEnabled);
```

Comment:
Crashing here if all tooltips are disabled isn't great (previously the check in `underCursor` would have prevented this.  It would be better to use `FirstOrDefault` and then only set `ActorTooltipExtra` / `TooltipType` if it is non-null.